src/screens/SendSheet.js

```diff
@@ -1009,7 +1026,7 @@ export default function SendSheet(props) {
             onChangeNativeAmount={onChangeNativeAmount}
             onResetAssetSelection={onResetAssetSelection}
             selected={selected}
-            sendMaxBalance={sendMaxBalance}
+            sendMaxBalance={() => setMaxEnabled(!maxEnabled)}
```

Comment:
looks good, only question is about this line - does this need to be toggle vs just always setMaxEnabled(true) when `sendMaxBalance` is pressed (edge case scenario of what happens when you switch from native > nonnative input or vice versa hitting max on both, would the 2nd "max" end up disabling)

Reply:
yes nice catch ty